core/chains/evm/txmgr/txmgr.go

```diff
@@ -100,7 +101,7 @@ type Txm struct {
 	q                pg.Q
 	ethClient        evmclient.Client
 	config           Config
-	keyStore         KeyStore
+	keyStore         keystore.EthWrappedKeyStore
```

Comment:
this type should become `KeyStore[...]`

Reply:
most of these changes are all tied to a design solution I thought of, ill explain it in a call - but I see what you mean ( I think maybe I'm overthinking and complicating it)